crates/biome_css_analyze/src/lint/nursery/no_unknown_unit.rs

```diff
@@ -0,0 +1,195 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic, RuleSource};
+use biome_console::markup;
+use biome_css_syntax::{
+    AnyCssDimension, CssFunction, CssGenericProperty, CssQueryFeaturePlain, CssSyntaxKind,
+};
+use biome_rowan::{SyntaxNodeCast, TextRange};
+
+const RESOLUTION_MEDIA_FEATURE_NAMES: [&str; 3] =
+    ["resolution", "min-resolution", "max-resolution"];
+
+// Check if the value is a CSS hack used in Internet Explorer.
```

Comment:
Internet explorer is a dead browser, we don't want to support it :)

Reply:
Understood. I have removed this process.